modules/services/sql/rds.tf

```diff
@@ -66,6 +67,26 @@ resource aws_db_instance mysql_rds_replicas {
   skip_final_snapshot = true
 
   max_allocated_storage = var.max_allocated_storage
+  parameter_group_name = aws_db_parameter_group.uq_daas_replica.name
+}
+
+resource "aws_db_parameter_group" "uq_daas_replica" {
+  name        = "uq-replicas-parameters"
+  description = "Extra parameters required for DaaS team to use Qlic tool with replicas"
+  family      = "mysql5.7"
+
+  parameter {
+    name         = "binlog_checksum"
+    value        = "CRC32"
+  }
+  parameter {
+    name         = "binlog_format"
+    value        = "row"
+  }
+  parameter {
+    name         = "binlog_row_image"
+    value        = "full"
+  }
```

Comment:
@Cup0fCoffee Is this necessary to be explicitly defined here? All these values seem to be the default.

AWS Docs says

> RDS for MySQL parameters are set to the default values of the storage engine that you have selected. For more information about MySQL parameters, see the MySQL documentation

and MySQL docs:

* [binlog_checksum](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_checksum)
* [binlog_format](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_format)
* [binlog_row_image](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_format)